plasma_framework/migrations/7_tenderly_setup.js

```diff
@@ -0,0 +1,43 @@
+const { execSync } = require('child_process');
+
+module.exports = async (
+    // eslint-disable-next-line no-unused-vars
+    deployer,
+    _,
+    // eslint-disable-next-line no-unused-vars
+    [deployerAddress, maintainerAddress, authorityAddress],
+) => {
+    const tenderly = process.env.TENDERLY || false;
+
+    if (tenderly) {
+        const installationStatus = execSync('curl https://raw.githubusercontent.com/Tenderly/tenderly-cli/master/scripts/install-linux.sh | sh', { encoding: 'utf-8' }); // the default is 'buffer'
```

Comment:
What happens if Tenderly is already installed?

Reply:
It would print a message that it is already installed.